spring-cloud-task-core/src/main/java/org/springframework/cloud/task/annotation/Task.java

```diff
@@ -46,4 +40,8 @@
 	 */
 	public String value() default "";
 
+	/**
+	 * Establishes the timeout for the task.  The default is -1 (meaning no timeout).
+	 */
+	public String timeout() default "-1";
```

Comment:
Since we don't support it yet, can we remove this and add it as part of the timeout story?